webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala

```diff
@@ -43,16 +43,164 @@ import org.knora.webapi.messages.util.rdf._
 import org.knora.webapi.messages.v2.responder.ontologymessages.OntologyMetadataGetByProjectRequestV2
 import org.knora.webapi.messages.v2.responder.ontologymessages.OntologyMetadataV2
 import org.knora.webapi.messages.v2.responder.ontologymessages.ReadOntologyMetadataV2
-import org.knora.webapi.responders.ActorDeps
 import org.knora.webapi.responders.IriLocker
+import org.knora.webapi.responders.IriService
 import org.knora.webapi.responders.Responder
 import org.knora.webapi.store.cache.settings.CacheServiceSettings
+import org.knora.webapi.store.triplestore.api.TriplestoreService
+import org.knora.webapi.util.ZioHelper
 
 /**
  * Returns information about projects.
  */
-final case class ProjectsResponderADM(actorDeps: ActorDeps, cacheServiceSettings: CacheServiceSettings)
-    extends Responder(actorDeps)
+trait ProjectsResponderADM {
+
+  /**
+   * Gets all the projects and returns them as a [[ProjectADM]].
+   *
+   * @return all the projects as a [[ProjectADM]].
+   *
+   *         NotFoundException if no projects are found.
+   */
+  def projectsGetRequestADM(): Task[ProjectsGetResponseADM]
+
+  /**
+   * Gets the project with the given project IRI, shortname, shortcode or UUID and returns the information as a [[ProjectADM]].
+   *
+   * @param identifier the IRI, shortname, shortcode or UUID of the project.
```

Comment:
```suggestion
   * Gets the project with the given project IRI, shortname or shortcode and returns the information as a [[ProjectADM]].
   *
   * @param identifier the IRI, shortname or shortcode of the project.
```

There is no UUID identifier anymore. It was living only during the "make project IRI not a shortcode but UUID based" crusade. I just forgot to remove it from docstrings here and there or someone brought it back with bad conflicts merge. You can ignore this suggestion here, I can fix it in separate PR.